plugins/woocommerce/includes/abstracts/abstract-wc-session.php

```diff
@@ -19,7 +19,7 @@ abstract class WC_Session {
 	/**
 	 * Customer ID.
 	 *
-	 * @var int $_customer_id Customer ID.
+	 * @var string $_customer_id Customer ID.
```

Comment:
I went ahead and updated this documentation as all the usage I've seen is always a string - though it is a numeric string when the user is logged in.